Makefile

```diff
@@ -54,23 +54,23 @@ install-pinned-macos: _conda_check
 # Run default tests
 test:
 	set -e
-	snakemake solve_elec_networks --configfile config/test/config.electricity.yaml --rerun-triggers=mtime
-	snakemake --configfile config/test/config.overnight.yaml --rerun-triggers=mtime
-	snakemake --configfile config/test/config.myopic.yaml --rerun-triggers=mtime
-	snakemake make_summary_perfect --configfile config/test/config.perfect.yaml --rerun-triggers=mtime
-	snakemake --configfile config/test/config.scenarios.yaml --rerun-triggers=mtime -n
+	snakemake solve_elec_networks --configfile config/test/config.electricity.yaml
+	snakemake --configfile config/test/config.overnight.yaml
+	snakemake --configfile config/test/config.myopic.yaml
+	snakemake make_summary_perfect --configfile config/test/config.perfect.yaml
+	snakemake --configfile config/test/config.scenarios.yaml -n
```

Comment:
@fneum This is always committed by me unintentionally.  but I like it more this way to make the tests really test everything. would you be fine by keeping it?

Reply:
Yes, just means the tests might run a little longer, but that is fine.